input/fsh/BFDR/profilesMigratingFromVRCL/ConditionEclampsiaHypertension_new.fsh

```diff
@@ -0,0 +1,20 @@
+Profile: ConditionEclampsiaHypertensionNew
+Parent: USCoreConditionProblemsHealthConcernsProfile
+Title: "Condition - Eclampsia Hypertension Vital Records"
+Description: "This Condition profile indicates that the mother had a pregnancy risk factor of eclampsia hypertension."
+* ^meta.versionId = "2"
+* ^meta.lastUpdated = "2023-04-19T02:09:59.117+00:00"
+* ^meta.source = "#tAYxo417PTp2yn2p"
+* ^version = "1.0.0"
+* ^experimental = false
+* ^publisher = "HL7 Public Health Working Group"
+* ^contact.name = "HL7 Public Health Working Group"
+* ^contact.telecom[0].system = #url
+* ^contact.telecom[=].value = "http://www.hl7.org/Special/committees/pher/index.cfm"
+* ^contact.telecom[+].system = #email
+* ^contact.telecom[=].value = "pher@lists.HL7.org"
+* ^jurisdiction = urn:iso:std:iso:3166#US "United States of America"
```

Comment:
I think Tricia said this, but you can delete all of this stuff...Sushi adds it for free.